astropy/coordinates/transformations.py

```diff
@@ -347,23 +347,34 @@ def get_transform(self, fromsys, tosys):
             self._composite_cache[fttuple] = comptrans
         return self._composite_cache[fttuple]
 
-    def lookup_name(self, name):
+    def lookup_name(self, name, raise_exception=False):
         """
         Tries to locate the coordinate class with the provided alias.
 
         Parameters
         ----------
         name : str
             The alias to look up.
-
+        raise_exception : Bool (default value is False)
+            False : doesn't raise exception if 'name' isn't found. (returns None)
+            True : raises exception if 'name' isn't found.
         Returns
         -------
         coordcls
+        if raise_exception == False:
```

Comment:
How does this look in the sphinx build?

Reply:
I was very uncomfortable with this, too, we don't do this format anywhere else. 

There is a sphinx section for exceptions called ``Raises``, I think that would be a good place to add this info.

```
Raises
------

KeyError
    If the coordinate class corresponding to the ``name`` does not exists
    and ``nameofnewargument`` is True.
```